src/Assistants/Models/StepwisePlannerSettings.cs

```diff
@@ -0,0 +1,13 @@
+﻿using YamlDotNet.Serialization;
+
+namespace SemanticKernel.Assistants.Models
+{
+    internal class StepwisePlannerSettings
+    {
+        [YamlMember(Alias = "max_iterations")]
+        public int MaxIterations { get; set; } = 15;
```

Comment:
maybe those default values could be set up all together in a static wellknown file, so they are easier to access and understand globally.